subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -208,6 +215,14 @@ struct node_rx_pdu {
 enum {
 	EVENT_DONE_EXTRA_TYPE_NONE,
 	EVENT_DONE_EXTRA_TYPE_CONN,
+/* Following proprietary defines must be at end of enum range */
+#if defined(CONFIG_BT_CTLR_USER_EXT)
+	EVENT_DONE_EXTRA_TYPE_PROPRIETARY_START,
```

Comment:
Can we be consistent and use `USER_START` and `USER_END`?

Reply:
Implemented suggested change